docs/deepops/dgx-diagnostic-firmware.md

```diff
@@ -0,0 +1 @@
+/home/atetelman/repo/sa/deepops/roles/nvidia-dgx-firmware/README.md
```

Comment:
Please create a link or a more substantive file with a pointer to the role README.

Reply:
You don't like just having a redirect/symlink?